sirepo/sim_data/__init__.py

```diff
@@ -179,6 +184,13 @@ def compute_model(cls, model_or_data):
         #TODO(robnagler) is this necesary since m is parsed?
         return cls.parse_model(cls._compute_model(m, d))
 
+    @classmethod
+    def dot_local_path(cls, subdir):
+        v = set(('src', 'share'))
+        assert subdir in v, \
+            f'subdir {subdir} not known {v}'
+        return pkio.py_path(f'/home/vagrant/.local/{subdir}/').join(cls.sim_type())
```

Comment:
I don't understand why this wouldn't be a configuration variable or even finding in path.

Reply:
https://github.com/radiasoft/sirepo/pull/3266/commits/2b4b6559d8022c36e74cdadc103a85f965e9927d